Gemfile

```diff
@@ -1,3 +1,7 @@
 source 'https://rubygems.org'
 
 gemspec
+
+if RUBY_VERSION >= '3.0.0'
```

Comment:
While this code is working, the rigorous way of comparing versions is by using [`Gem::Version`](https://ruby-doc.org/stdlib-2.7.0/libdoc/rubygems/rdoc/Gem/Version.html).

Check this out:

```ruby
2.7.2 :001 > '3.0' > '10.0'
 => true 
2.7.2 :002 > Gem::Version.new('3.0') > Gem::Version.new('10.0')
 => false
```

:grin: 

When using the Ruby version, there's a ready API:

```ruby
2.7.2 :003 > Gem.ruby_version > Gem::Version.new('10.0')
 => false
```

Although the bad behavior will likely never be triggered, I place much value on exact semantics/logic :smile:.

Reply:
Thank you for reviewing!
I changed way to check Ruby version according to your proposal.